res_partner_operating_unit/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# © 2017 Niaga Solution - Edi Santoso <repodevs@gmail.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = 'res.partner'
+
+    operating_unit_id = fields.Many2one(
```

Comment:
@Dangar-Serpentcs Can you change the type to a many2many field and adapt the record rules?

The default value should remain the default OU from the user.

Reply:
Sure